partcad/src/partcad/test/test.py

```
@@ -0,0 +1,57 @@
+
+    @abstractmethod
+    async def test(self, tests_to_run: list["Test"], ctx, shape, test_ctx: dict = {}) -> bool:
+        raise NotImplementedError("This method should be overridden")
+
+    async def test_log_wrapper(self, tests_to_run: list["Test"], ctx, shape, test_ctx: dict = {}) -> bool:
+        # shape_name = shape.config["name"] if "name" in shape.config else shape.name
+        test_ctx = copy.copy(test_ctx)
+        test_ctx["log_wrapper"] = True
+        action_name = (
+            shape.project_name
+            if not "action_prefix" in test_ctx
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)